Cargo.toml

```diff
@@ -31,6 +31,7 @@ itertools = "0.11.0"
 tempfile = "3.10.1"
 regex = "1.10"
 lazy_static = "1.4.0"
+indoc = "2"
```

Comment:
What do you need it for?

Reply:
I added this so it is easier for me to test missing libfuncs from versioned constants file. It's added to dev-dependencies.